examples/outputs/recursive_alias.rs

```diff
@@ -0,0 +1,217 @@
+    dotenv().ok();
+
+    let node_url = env::var("NODE_URL").unwrap();
+    let faucet_url = env::var("FAUCET_URL").unwrap();
+
+    // Create a client instance.
+    let client = Client::builder()
+        .with_node(&node_url)?
+        .with_node_sync_disabled()
+        .finish()?;
+
+    let secret_manager = SecretManager::Mnemonic(MnemonicSecretManager::try_from_mnemonic(
+        &env::var("NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1").unwrap(),
+    )?);
+
+    let address = client.get_addresses(&secret_manager).with_range(0..1).get_raw().await?[0];
+    println!(
+        "{}",
+        request_funds_from_faucet(&faucet_url, &address.to_bech32(client.get_bech32_hrp().await?)).await?
+    );
+    // Wait some time for the faucet transaction
+    tokio::time::sleep(std::time::Duration::from_secs(15)).await;
+
+    let rent_structure = client.get_rent_structure().await?;
+
+    //////////////////////////////////
+    // create three new alias outputs
+    //////////////////////////////////
+    let alias_output_builder =
+        AliasOutputBuilder::new_with_minimum_storage_deposit(rent_structure.clone(), AliasId::null())?
+            .add_feature(Feature::Sender(SenderFeature::new(address)))
+            .with_state_metadata(vec![1, 2, 3])
+            .add_immutable_feature(Feature::Issuer(IssuerFeature::new(address)))
+            .add_unlock_condition(UnlockCondition::StateControllerAddress(
+                StateControllerAddressUnlockCondition::new(address),
+            ))
+            .add_unlock_condition(UnlockCondition::GovernorAddress(GovernorAddressUnlockCondition::new(
+                address,
+            )));
+
+    let outputs = vec![alias_output_builder.clone().finish_output()?; 3];
+
+    let block_1 = client
+        .block()
+        .with_secret_manager(&secret_manager)
+        .with_outputs(outputs)?
+        .finish()
+        .await?;
+
+    println!(
+        "Transaction with new alias outputs sent: {node_url}/api/core/v2/blocks/{}",
+        block_1.id()
+    );
+    let _ = client.retry_until_included(&block_1.id(), None, None).await?;
+
+    //////////////////////////////////
+    // create second transaction with the actual AliasId (BLAKE2b-160 hash of the Output ID that created the alias) and
+    // make both alias outputs controlled by the first one
+    //////////////////////////////////
+    let alias_output_ids = get_new_alias_output_ids(block_1.payload().unwrap())?;
+    let alias_id_0 = AliasId::from(alias_output_ids[0]);
+    let alias_id_1 = AliasId::from(alias_output_ids[1]);
+    let alias_id_2 = AliasId::from(alias_output_ids[2]);
+
+    let alias_0_address = Address::Alias(AliasAddress::new(alias_id_0));
+    let alias_1_address = Address::Alias(AliasAddress::new(alias_id_1));
+
+    let outputs = vec![
+        // make second alias output be contolled by the first one
+        alias_output_builder
+            .clone()
+            .with_alias_id(alias_id_1)
+            .with_state_index(0)
```

Comment:
```suggestion
            .clone()
            .with_alias_id(alias_id_1)
            .replace_feature(Feature::Sender(SenderFeature::new(alias_0_address)))?
            .with_state_index(0)
```

This should work, right? But if I try that, I'm getting `Error: TransactionSemantic(UnverifiedSender)`. I'm guessing the automatic selection isn't working yet for features.

Reply:
@PhilippGackstatter I noticed that issue yesterday evening,fixing it atm

Reply:
Should work now, let me know if you find other issues

Reply:
Works on my end, thanks for the quick fix!